File: study/kotlin_study/src/main/java/com/pds/kotlin/study/net/ApiManager.java
package com.pds.kotlin.study.net;

import com.blog.pds.net.RetrofitProvide;
import com.pds.env.med.PatientEnv;

/**
 * @author: pengdaosong
 * @CreateTime: 2020/9/23 5:44 PM
 * @Email: pengdaosong@medlinker.com
 * @Description:
 */
public enum ApiManager {

    INSTANCE;

    private static AppApi sAppApi;

    /**
     * 用户相关数据Api
     */
    public static AppApi getApi() {
        if (sAppApi == null) {
            sAppApi = RetrofitProvide.getInstance()
                    .build(RetrofitProvide.getInstance().buildRetrofit2(PatientEnv.getAppEnv()),
                            AppApi.class);
        }
        return sAppApi;
    }
}

File: IM/socket_control/src/main/java/blog/pds/com/socket/control/config/ApiManager.kt
package blog.pds.com.socket.control.config

/**
 * @author: pengdaosong
 * CreateTime:  2019-09-04 17:01
 * Email：pengdaosong@medlinker.com
 * Description:
 */

object ApiManager{
    var ip: String? = ""
    var port: Int? = 0

    fun ipAndPort(ip: String?,port: Int?){
        this.ip = ip
        this.port = port
    }
}